gtk4-macros/src/attribute_parser.rs

```diff
@@ -273,9 +273,28 @@ pub fn parse_fields(fields: &Fields) -> Result<Vec<AttributedField>, Error> {
     let mut attributed_fields = Vec::new();
 
     for field in fields {
+        let mut has_attr = false;
         if !field.attrs.is_empty() {
             if let Some(attributed_field) = parse_field(field)? {
-                attributed_fields.push(attributed_field)
+                attributed_fields.push(attributed_field);
+                has_attr = true;
+            }
+        }
+        if !has_attr {
+            if let syn::Type::Path(syn::TypePath { path, .. }) = &field.ty {
+                if path.leading_colon.is_none()
+                    && path.segments.len() == 1
+                    && matches!(
+                        &path.segments[0].arguments,
+                        syn::PathArguments::AngleBracketed(_)
+                    )
+                    && path.segments[0].ident == "TemplateChild"
+                {
+                    proc_macro_error::emit_warning!(
```

Comment:
If you don't want to make this an error you probably want to add some kind of `#[allow(gtk4::template_child_without_attribute)]` or similar to silence it

Reply:
Works now with `#[template(file = "...", allow_template_child_without_attribute)]`